### ISSUE TITLE
Allow to reset Apple Health permissions

### ISSUE THREAD

**Issue by SergejLogis** [AvdLee/RocketSimApp#374]
**Is your feature request related to a problem? Please describe.**
When developing Apple Health-related features it would be great to reset granted/denied permissions without reinstallation of an app.

**Describe the solution you'd like**
Add Apple Health to the `Grant/Revoke/Reset Permissions` menu.

**Describe alternatives you've considered**
I've tried to reset all permissions, but Apple Health wasn't reset.

**Additional context**
None.

**Comment by AvdLee** [comment]
@SergejLogis yes, great idea! Unfortunately, I'm dependent on what `xcrun simctl privacy --help` provides. I'll keep this issue open, but it would be best to start with a feedback report to Apple for requesting this as a feature for `simctl`

**Comment by AvdLee** [comment]
Found a potential solution, so we can bring this back in the roadmap!
